README.md

```diff
@@ -11,6 +11,8 @@ Material Components for the web is the successor to [Material Design Lite](https
 
 MDC-Web strives to seamlessly incorporate into a wider range of usage contexts, from simple static websites to complex, JavaScript-heavy applications to hybrid client/server rendering systems. In short, whether you're already heavily invested in another framework or not, it should be easy to incorporate Material Components into your site in a lightweight, idiomatic fashion.
 
+> **Note:** MDC-Web is intended to be used for other developers to execute Material Design properly. MDC-Web delegates higher level capabilities to other JS frameworks and libraries. An example is that MDC-Web neither use web components nor renders the DOM of its components. By doing low-level operations, MDC-Web its able to accomplish its goals.
```

Comment:
We want to encourage _both_:

- People who just want to easily give a website a Material Design look and feel can drop MDC-Web onto their page and use our convenience mechanisms to instantiate and manage the components

- People who need more control over components can use them a-la-carte and integrate them into their frameworks, etc.